hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -122,10 +122,18 @@
     {
       name: build_seed
       pre_build_cmds: [
-        '''cd {proj_root} && ./util/topgen.py -t {ral_spec} \
-               -o hw/top_earlgrey --rnd_cnst_seed {build_seed}
-        ''',
-        "cd {proj_root} && ./util/design/gen-otp-mmap.py --seed {build_seed}"
+        // TODO: find where in run phase we are using this pkg. It fails during ibex TLUL integrity
+        // check.
+        // '''cd {proj_root} && ./util/topgen.py -t {ral_spec} \
+        //        -o hw/top_earlgrey --rnd_cnst_seed {seed}
+        // ''',
```

Comment:
This command only fails when using the EDACloud launcher. If using local workstation to run, there isn't any issue. 
So I wonder if it relates to some script or backdoor load during run time using the old auto-generated RTL file.
Here is the ECC error waves. I am debugging that thought to comment it out so at least nightly regression will run.
I will file an issue to track this TODO.
![image](https://user-images.githubusercontent.com/11466553/163635304-f00bda70-80f3-4f1a-ba34-20dc47295e5e.png)